helper/helper.c

```diff
@@ -62,7 +62,8 @@ void STANDBY_BlinkGreen(void)
 		gGreenLedTimer = 0;
 	}
 	if (gBlinkGreen && gGreenLedTimer > 199) {
-		if (!gScannerMode && gRadioMode != RADIO_MODE_RX) {
+		//if (!gScannerMode && gRadioMode != RADIO_MODE_RX) {
+		if ((!gScannerMode && gRadioMode != RADIO_MODE_RX) || (gScannerMode && !gExtendedSettings.ScanBlink)) {
```

Comment:
```suggestion
		if ((!gScannerMode || !gExtendedSettings.ScanBlink) && gRadioMode != RADIO_MODE_RX) {
```
This will prevent the LED from powered off when receiving during a scan
(no need to keep the previous condition in comment, git doing that for us)